gstreamer/src/message.rs

```diff
@@ -1165,14 +1189,7 @@ macro_rules! message_builder_generic_impl {
             }
         }
 
-        pub fn other_fields(self, other_fields: &[(&'a str, &'a ToSendValue)]) -> Self {
```

Comment:
For the 0.10 branch this should stay, do best to remove it in a separate commit in this PR